.github/workflows/main.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         export VITE_BASE_DIR="/$(basename $GITHUB_REPOSITORY)/"
         echo "VITE_BASE_DIR: $VITE_BASE_DIR"
         ./run.sh
+    - name: Predeployment test
```

Comment:
Let's not do run.sh twice.

Could we instead try to split the jobs here and keep it within this workflow file to avoid redundancy?